vespajlib/src/test/java/com/yahoo/compress/ZstdCompressorTest.java

```diff
@@ -0,0 +1,22 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.compress;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+/**
+ * @author bjorncs
+ */
+class ZstdCompressorTest {
+
+    @Test
+    void compresses_and_decompresses_input() {
+        byte[] inputData = "The quick brown fox jumps over the lazy dog".getBytes();
+        ZstdCompressor compressor = new ZstdCompressor();
+        byte[] compressedData = compressor.compress(inputData, 0, inputData.length);
+        byte[] decompressedData = compressor.decompress(compressedData, 0, compressedData.length);
```

Comment:
You should check that the data has actually been compressed. Simplest is just to check that the output is shorter than the input.